packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -469,16 +471,39 @@ export class Resolver {
           isAddPendingDocument,
         })
       }
-      if (isDeletion) {
+      // if we are deleting a document or updating its name we should check if it exists
+      if (isDeletion || isUpdateName) {
         if (!alreadyExists) {
           throw new Error(
             `Unable to delete document, ${realPath} does not exist`
```

Comment:
This error message should reflect the action that is failing (ie. `delete` or `update`)

Reply:
Makes sense!